.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Cmdlet(s)
+
+### PowerShell Version
+
+### Module Version
+
+### OS Version
+
+### Description
+
+### Debug Output
```

Comment:
Instructions: to get DEbug Output, set $Debugpreference="Continue" and then execute the cmdlet or script causing the issue